src/utils/allocators.zig

```diff
@@ -11,7 +11,7 @@ pub fn RecycleFBA(config: struct {
         // (only used on init/deinit + arraylist expansion)
         backing_allocator: std.mem.Allocator,
         // this does the data allocations (data is returned from alloc)
-        alloc_allocator: std.heap.FixedBufferAllocator,
+        fba_allocator: std.heap.FixedBufferAllocator,
```

Comment:
The previous name was silly 😆 . You can name this one `fba`, it already has "allocator" in the name.

Reply:
even though the 'a' in 'fba' is allocator - i still prefer the explicit '_allocator' because its just so clear that its an allocator

Reply:
fair enough, no problem from me